orchestra/tasks.py

```diff
@@ -0,0 +1,369 @@
+from datetime import datetime
+from logging import getLogger
+from pathlib import Path
+
+import dill
+import luigi
+import yaml
+from luigi.util import inherits, requires
+
+from epimodel import Level, RegionDataset, algorithms, imports, read_csv_smart, utils
+from epimodel.exports.epidemics_org import process_export, upload_export
+from epimodel.gleam import Batch, GleamDefinition, batch as batch_module
+
+log = getLogger(__name__)
+
+
+def default_from_config(task_name: str, param_name: str) -> dict:
+    return dict(section=task_name, name=param_name)
+
+
+class RegionsFile(luigi.ExternalTask):
+    # TODO: we can use config_path to fetch defaults from the luigi.cfg
+    # TODO: or use the default in parameter. Now we use both for different tasks
+    regions = luigi.Parameter(default="data/regions.csv")
+
+    def output(self):
+        return luigi.LocalTarget(self.regions)
+
+
+class GleamRegions(luigi.ExternalTask):
+    gleams = luigi.Parameter(default="data/regions-gleam.csv")
+
+    def output(self):
+        return luigi.LocalTarget(self.gleams)
+
+
+@inherits(RegionsFile, GleamRegions)
+class RegionsDatasetTask(luigi.Task):
```

Comment:
This is my way of dealing with the fact that luigi tasks can't pass between each other anything else but files/parameters. I create the `rds`, dump it to disk and then load where needed, explicitly stating this is the upstream task